File: pset6/sentimental-mario-less/mario.py
# creates a pyramid-like structure based on user input
from cs50 import get_int

# prompts user for a height between 1 and 8
while True:
    height = get_int("Enter height ")
    if height > 0 and height < 9:
        break

# prints the pyramid
for i in range(height):
    for j in range(i):
        print("#", end='')
    print("#")

File: pset1/mario-less/mario.c
//creates a pyramid-like structure with a height based on the user's input
#include <cs50.h>
#include <stdio.h>

int main(void)
{

//input of "height" sets the exact height for the pyramid with a a minimum of 1 and a max of 8

    int height;

    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

//prints a pyramid consisting of spaces and hashes (#)
    for (int i = 0; i < height; i++)
    {
        for (int j = i; j < height - 1; j++)
        {
            printf(" ");
        }
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }
        printf("\n");
    }


}